index.html

```diff
@@ -61,14 +61,15 @@ <h2 id="general">General Information</h2>
   the pitch.
 -->
 <p>
-  <a href="{{site.swc_site}}">Software Carpentry</a>'s mission
-  is to help scientists and engineers get more research done in less
-  time and with less pain by teaching them basic lab skills for
-  scientific computing.  This hands-on workshop will cover basic
-  concepts and tools, including program design, version control, data
-  management, and task automation.  Participants will be encouraged to
-  help one another and to apply what they have learned to their own
-  research problems.
+  <a href="{{site.swc_site}}">Software Carpentry</a>
+  teaches researchers how to get their work done
```

Comment:
How about

"helps researchers get their work done
  in less time and with less pain
  by teaching them basic research computing skills."

(even better perhaps would be 'SWC aims to help researchers'